GDAXSharp/Shared/Types/Side.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GDAXSharp.Shared.Types
+{
+    public enum Side
```

Comment:
this already exists here https://github.com/dougdellolio/gdax-csharp/blob/master/GDAXSharp/Services/Orders/Types/OrderSide.cs

Reply:
Shouldn't this one be in the `Shared` folder? Because multiple services use this enum and not all of them are of the type `Order`